### PR TITLE
6.2.33 beta 2

### PR DESCRIPTION
**Pull request by ikedas**
Updates according to changes on upstream:
  - \#170 WWSympa: Switch to Foundation 6
  - \#220 static_content directory structure
  - \#336 Starting a test framework

### CODE REVIEW

File: sympa.spec [hunk 1]
```diff
@@ -76,6 +75,7 @@ BuildRequires: perl(Cwd)
 BuildRequires: perl(Data::Password)
 BuildRequires: perl(DateTime)
 BuildRequires: perl(DateTime::Format::Mail)
+BuildRequires: perl(DBD::SQLite)
```
Comment (xavierba) [inline sympa.spec:78]: I guess this is a new requirement from the test suite ?

File: sympa.spec [hunk 2]
```diff
@@ -207,19 +207,6 @@ Requires:    impallari-raleway-fonts
...
-%else
-Provides:    bundled(js-foundation) = 5.5.3
-# Bundled in bundled js-foundation
-Provides:    bundled(js-fastclick)
-Provides:    bundled(js-jquery-cookie) = 1.4.1
-Provides:    bundled(js-jquery-requestAnimationFrame)
-Provides:    bundled(js-modernizr) = 2.8.3
-Provides:    bundled(js-placeholder) = 2.0.9
-Provides:    bundled(js-sizzle) = 2.2.0
-%endif
```
Comment (xavierba) [inline sympa.spec]: This part shouldn't be removed but updated for foundation 6:
```
%if %{unbundle_foundation}
Requires:    js-foundation6
%else
Provides:    bundled(js-foundation) = 6.4.2
# Bundled in bundled js-foundation
Provides:    bundled(js-what-input) = 4.2.0
%endif
```

File: sympa.spec (hunk 1 above)
Comment (ikedas) [reply sympa.spec:78]: Yes, it is used by a new test `t/02_Sympa_DatabaseManager.t` in sympa-community/sympa#336

File: sympa.spec (hunk 2 above)
Comment (ikedas) [reply sympa.spec]: I repushed commits according to your suggestion.

File: sympa.spec [hunk 3]
```diff
@@ -14,13 +14,12 @@
...
-%global unbundle_modernizr      0
 %global unbundle_respond        0%{?fedora}%{?rhel}
 
-#global pre_rel b.3
+%global pre_rel b.2
 
 Name:        sympa
-Version:     6.2.32
+Version:     6.2.33
 Release:     %{?pre_rel:0.}2%{?pre_rel:.%pre_rel}%{?dist}
```
Comment (xavierba) [inline sympa.spec]: Nitpick, the '2' in the release field above should be '1' and the corresponding change should be applied to the changelog entry.
The release field needs to be reset to 1 (or 0.1 for a pre-release, but here the macros in the release field take care of the leading '0.' automagically) each time the version field is incremented (and also when the software goes from a pre-release to the final release, but this is not the case with Sympa, as next stable will be 6.2.34).

**Comment by ikedas** [comment]
I understood. I re-pushed.

**Pull request merged**